### DISCUSSION TITLE
update with era5L before 1981

### DISCUSSION THREAD

**Discussion by jnthnroy** [kratzert/Caravan#14] [Q&A]
Hi,
Thank for sharing all this work, it is a great rep!
It is planned to update the availlable history with era5L before 1981?

**Comment by kratzert** [comment]
Yes, this is planned but we need to wait for Earth Engine to ingest the full archive of ERA5-Land data. Once the full record is available, we will most likely add this period as an extension.

**Comment by jonschwenk** [comment]
As of 4/12/23, [ERA5-Land **daily** aggregates](https://developers.google.com/earth-engine/datasets/catalog/ECMWF_ERA5_LAND_DAILY_RAW) are available from 1963 (weird?), and [monthly ](https://developers.google.com/earth-engine/datasets/catalog/ECMWF_ERA5_LAND_MONTHLY_AGGR)have been ingested back to 1950. Hourly are still stuck at 1981.

**Reply by kratzert** [reply]
Sorry for the late reply, I had and accident which required surgery and wasn't working for the past weeks. Right now I am at the EGU so still less at the PC. Hourly data is also available from before 1981, even if the data catalog says otherwise. Just use the code snippet from the hourly page and change dates to e.g. 1970. I have no insights here but at some point all data back to 1950 should be available.

**Comment by jonschwenk** [comment]
Ah, sorry to hear that! Thanks for the update--I'll ignore the metadata and
see what I get. Be well!
